web-components/src/components/inputs/countries.ts

```diff
@@ -0,0 +1,222 @@
+export const countries: { [key: string]: string } = {
```

Comment:
it seems like we have this file twice now, both in web-components and web-registry
could you remove it from web-registry and refactor the components that use it to use the new `LocationStateField` and `LocationCountryField` components (see `BuyCreditsModal` and `CreditsPurchaseForm`)?

Reply:
unresolving, seems like we're still not using the new `LocationStateField` and `LocationCountryField` components in ` BuyCreditsModal` and `CreditsPurchaseForm`